src/memphis.ts

```diff
@@ -164,6 +176,59 @@ export class Memphis {
         });
     }
 
+    private async _scemaUpdatesListener(stationName: string, schemaUpdateData: Object): Promise<void> {
+        try {
+            let empty = schemaUpdateData['schema_name'] === '';
+            const subName = stationName.replace(/\./g, '#');
+            let schemaDataExist = this.stationSchemaDataMap.has(subName);
+            if (empty) {
+                this.stationSchemaDataMap.set(subName, {});
+                this.meassageDescriptors.set(subName, null);
+            } else {
+                let protoPathName = `${__dirname}/memphisMeassageDescriptor_${schemaUpdateData['schema_name']}.proto`;
```

Comment:
this is not message descriptor, and in order to avoid 2 creations of files with the same name please call this file <schema_name>_<version>.proto